api/app/controllers/api/v1/projects_controller.rb

```diff
@@ -27,6 +29,7 @@ def index
       def show
         @scope_for_projects = Project.friendly
         @project = load_project
+        authorize_action_for @project if @project.draft?
```

Comment:
Remove the condition — as above, the `if @project.draft?` is actually authorization logic that doesn't belong in this controller. Imagine if we has another controller that returned projects. We'd have to repeat this logic for every controller that touched projects. This PR is a good example of why DRY is about more than typing less and saving time — DRY code is more flexible and easier to refactor.